src/pages/studyView/table/CNAGenesTable.tsx

```diff
@@ -20,6 +23,9 @@ class CNAGenesTableComponent extends LazyMobXTable<CopyNumberCountByGene> {
 
 @observer
 export class CNAGenesTable extends React.Component<ICNAGenesTablePros, {}> {
+    @computed get data() {
```

Comment:
we don't need to access this via computed property. basically, the result property should be dereferenced in the same place as where its status is checked, so that the relationship between the two is made obvious.  as it stands now, a developer would have know idea they need to check status before referening the data property.   Alternatively, the status can be dealt with by the parent of this component and the dereferenced data passed directly to the component.  That way the component doesn't need to have knowledge of status because the data will always be populated.

Reply:
But I can see you are checking status in the render method, so just reference result directly there.